linker/shared/src/main/scala/org/scalajs/linker/backend/emitter/FunctionEmitter.scala

```diff
@@ -894,10 +909,43 @@ private[emitter] class FunctionEmitter(sjsGen: SJSGen) {
             implicit val env = env0
             val jsArgs = newArgs.map(transformExprNoChar(_))
 
-            if (es2015)
-              js.Apply(js.DotSelect(jsArgs.head, js.Ident("copyTo")), jsArgs.tail)
-            else
-              genCallHelper("systemArraycopy", jsArgs: _*)
+            def withoutChecks: js.Tree = {
+              if (es2015)
+                js.Apply(js.DotSelect(jsArgs.head, js.Ident("copyTo")), jsArgs.tail)
+              else
+                genCallHelper("systemArraycopy", jsArgs: _*)
+            }
```

Comment:
IMO, it is still meaningful. Or at least, it is no less meaningful now than before this PR. It is useful to reduce the complexity of code generation here. Otherwise we would have to translate differently the arguments 0 and 2 than the other ones.

Reply:
> Otherwise we would have to translate differently the arguments 0 and 2 than the other ones.

Yes, fair enough, I missed that.

> This appears here and in the CoreJSLib . Put a helper (uncheckedArrayCopy?) in SJSGen?

IMO this still might have value. Or you don't think so?

Reply:
> IMO this still might have value. Or you don't think so?

Yes, indeed. I had overlooked that comment. Done.